kite-data/kite-data-hbase/src/main/java/org/kitesdk/data/hbase/HBaseMetadataProvider.java

```diff
@@ -168,6 +171,9 @@ public DatasetDescriptor update(String namespace, String name, DatasetDescriptor
     } else {
       LOG.info("Schema hasn't changed, not migrating: (" + name + ")");
     }
+
+    Map<String, String> properties = getProperties(descriptor);
```

Comment:
Do you think this should be a descriptor method? `descriptor.getProperties()` would make sense, along with a setter on the descriptor builder.